### PR TITLE
[iOS] install ios-deploy in correct way & add missing CocoaPods tools…

### PR DESCRIPTION
**Pull request by mosabab**
[iOS] install ios-deploy in correct way & add missing information regarding CocoaPods tools that needed to build iOS apps.

### Platforms affected
iOS

### Motivation and Context

- How to install ios-deploy tool in correct way.
- Add missing information about CocoaPods tools that needed to build iOS apps.

### Checklist

- [x] I've run the tests to see all new and existing tests pass
- [x] I added automated test coverage as appropriate for this change
- [x] Commit is prefixed with `(platform)` if this change only applies to one platform (e.g. `(android)`)
- [x] If this Pull Request resolves an issue, I linked to the issue in the text above (and used the correct [keyword to close issues using keywords](https://help.github.com/articles/closing-issues-using-keywords/))
- [x] I've updated the documentation if necessary

### CODE REVIEW

File: www/docs/en/dev/guide/platforms/ios/index.md [hunk 1]
```diff
@@ -69,10 +69,20 @@ $ xcode-select --install
 The [ios-deploy](https://www.npmjs.org/package/ios-deploy) tools allow you
 to launch iOS apps on an iOS Device from the command-line.
 
-To install it, run the following from command-line terminal:
+Install ios-deploy via [Homebrew](https://brew.sh/) by running:
 
 ```bash
-$ npm install -g ios-deploy
+$ brew install ios-deploy
```
Comment (breautek) [inline www/docs/en/dev/guide/platforms/ios/index.md:75]: What's wrong with installing `ios-deploy` from npm?

Comment (mosabab) [reply www/docs/en/dev/guide/platforms/ios/index.md:75]: On macOS there is an error if you try to install it from npm.

According to the main page of ios-deploy, please refer to new installation guide:
https://github.com/ios-control/ios-deploy#Installation

Comment (mosabab) [reply www/docs/en/dev/guide/platforms/ios/index.md:75]: And also you can refer to installation guide at npm main page for the ios-deolpy:
https://www.npmjs.com/package/ios-deploy#installation

Comment (erisu) [reply www/docs/en/dev/guide/platforms/ios/index.md:75]:
```
$ brew info ios-deploy
ios-deploy: stable 1.10.0 (bottled), HEAD
```
`brew` has version `1.10.0` released.

```
$ npm info ios-deploy
ios-deploy@1.11.0 | GPLv3 | deps: none | versions: 41
```
`npm` has version `1.11.0` released.

Just for information, `npm` is newer.

**Comment by breautek** [comment]
I think this PR should be actually two separate PRs.

One change for the CocoaPods update, the other for the ios-deploy update.

**Comment by mosabab** [comment]
> I think this PR should be actually two separate PRs.
> 
> One change for the CocoaPods update, the other for the ios-deploy update. Otherwise I think they are good.

I was closed the separated 2 PRs in the following:
https://github.com/apache/cordova-docs/pull/1095
https://github.com/apache/cordova-docs/pull/1102

I can close this one and reopen the 2 closed, but it same information.

**Comment by breautek** [comment]
https://github.com/apache/cordova-docs/pull/1095 appears to be a copy of this one?

https://github.com/apache/cordova-docs/pull/1102/files looks good as is, I'll reopen that one.

**Comment by mosabab** [comment]
> #1095 appears to be a copy of this one?
> 
> https://github.com/apache/cordova-docs/pull/1102/files looks good as is, I'll reopen that one.
> 
> edit: err I can't reopen the PR because the remote has been deleted.

I will make new 2 separated PR for this .

**Comment by mosabab** [comment]
Close this one.

And open 2 new separated PR:
https://github.com/apache/cordova-docs/pull/1104
https://github.com/apache/cordova-docs/pull/1105

**Pull request closed without merging**